tests/app/test_frontend_proxy.py

```diff
@@ -46,9 +46,7 @@ def test_listen_http_with_presence_enabled(self):
         # Grab the resource from the site that was told to listen
         self.assertEqual(len(self.reactor.tcpServers), 1)
         site = self.reactor.tcpServers[0][1]
-        self.resource = (
-            site.resource.children[b"_matrix"].children[b"client"].children[b"r0"]
-        )
+        self.resource = site.resource.children[b"_matrix"].children[b"client"]
```

Comment:
Should this change be reflected somewhere else? Seems odd to just lop off `r0` here.

Reply:
This is just a side effect of whether we register the `JsonResource` at `/_matrix/client/r0` or `/_matrix/client` in the code. For whatever reason the frontend proxy was registering the resource at `/_matrix/client/r0`, while everything else uses `/_matrix/client`

Reply:
Ah ok, so just fixing an inconsistency. Cool :)